### DISCUSSION TITLE
PDOCrud

### DISCUSSION THREAD

**Discussion by DOESAdmin** [community/community#41030] [New to GitHub]
### Select Topic Area

Show & Tell

### Body

Good day Gitters

I was wondering if there is anyone that could perhaps help me out, I am aware that it isn't a Git product, but if you have any knowledge about this product, it would be highly appreciated if you could guide me to get it installed please. I purchased it on Envato and have not yet heard back from the author. Please help me out with this.

Kind regards

**Comment by ghostinhershell** [comment]
Welcome to the GitHub Community, @DOESAdmin, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
